File: easy/pro-balconies/Balconies.c
#include <stdio.h>

int main() {
    int aw, al, bw, bl;
    scanf("%d,%d", &aw, &al);
    scanf("%d,%d", &bw, &bl);
    
    char ap = 'A' + (aw * al < bw * bl);
    printf("Apartment %c", ap);
    
    return 0;
}

File: easy/pro-balconies/Balconies.rb
aWidth, aLength = gets.chomp.split(",")
bWidth, bLength = gets.chomp.split(",")

isBiggerA = aWidth.to_i * aLength.to_i > bWidth.to_i * bLength.to_i

puts "Apartment " + (isBiggerA ? "A": "B")
